ruby/data.rb

```diff
@@ -6,28 +6,23 @@
 		Assumption: This task will run against the webserver, so that it's able to fetch the database credentials
 		from _ss_environment.php
 
-		Example command: cap -f '/sites/deploynaut/www/assets/Capfile' project1:env1 data:getdb -s data_path=/tmp/mydatabase.sql
+		Example command: cap -f '/sites/deploynaut/www/assets/Capfile' project1:env1 data:getdb -s data_path=/tmp/mydatabase.sql.gz
 
 		Required arguments to the cap command:
-		data_path - Absolute path (including filename) of where the exported database should be placed, e.g. /tmp/my_database.sql
+		data_path - Absolute path (including filename) of where the exported database should be placed, e.g. /tmp/my_database.sql.gz
 	DESC
 	task :getdb do
-		# todo: output to gzip and stream that back instead of the raw data.
-		run "mysqldump --skip-opt --add-drop-table --extended-insert --create-options --quick --set-charset --default-character-set=utf8 #{mysql_options} -p", :once => true do |channel, stream, data|
+		tmpfile = "/tmp/temp-download-database.sql.gz"
```

Comment:
Do you think we should future proof it by creating a temporary race condition safe name like http://stackoverflow.com/questions/13787746/creating-a-thread-safe-temporary-file-name ?